api-examples/Input/README.md

```diff
@@ -0,0 +1,74 @@
+# &lt;Input /&gt;
+
+Text editing in widgets!
+
+<a href="https://www.loom.com/share/c7cf2450af744244a5728cb82aa28560">
+  <img style="max-width:300px;" src="https://cdn.loom.com/sessions/thumbnails/c7cf2450af744244a5728cb82aa28560-1645663821012-with-play.gif">
+  <p>Figma Widget API - Input Component - 23 February 2022 - Watch Video</p>
+</a>
+
+## Introduction
```

Comment:
Could you explain why this README is necessary? It might be easier to link out to the plugin docs instead? Otherwise we will have to maintain an extra page if the API changes.

Reply:
good call, this is just coming over from the old docs, i can remove.